packages/vite/src/template-tag.ts

```diff
@@ -1,15 +1,14 @@
 import { createFilter } from '@rollup/pluginutils';
 import type { Plugin } from 'vite';
-import { readFileSync } from 'fs';
 import { Preprocessor } from 'content-tag';
 
-const gjsFilter = createFilter('**/*.gjs?(\\?)*');
+const gjsFilter = createFilter('**/*.{gjs,gts}?(\\?)*');
```

Comment:
I'm not comfortable adding gts to this for now. I know we should deal with the Typescript situation but I feel like that should be a wholistic PR that explicitly adds support rather than changing this one place

Reply:
What is 'the typescript situation'?

For running ts code, babel (or whatever) removes the type annotations and that's it

Reply:
yes but my understanding is that at this level we should always only be seeing gjs files no? and by "situation" I mean fully end-to-end tests on it with typechecking, that should be a different PR in my opinion

Reply:
Right, as of now it would not make any difference. Since currently it's expected that the previous stage compiles the gts files.
It would work together with #1679 . Which would work for vite and then i could make it work in the tests in #1718

Reply:
Why wouldn't we see the input file ext, if gjs is here already?

If it were compiled, we'd just have js, yeah?

Reply:
Potentially could also work with webpack if content-tag is added as a loader. Or is there already one?
Why would it work with gjs but not gts?

Reply:
Also, .ts is already part of the entry points and vite can load it directly. E.g the vite-app is not setup to transform ts files for the previous stage.
But sure, i can move this gts part elsewhere.
Shall i add it to #1679 ?

Reply:
> Why wouldn't we see the input file ext, if gjs is here already?
> 
> If it were compiled, we'd just have js, yeah?

True, somehow i was thinking gts transforms to gjs first... But it would be js directly, right?

Reply:
Right, there is no gts to gjs conversion.
I think it goes: gts->ts->js

For content-tag -> babel/whatever